packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -66,14 +66,15 @@ class ExpandIcon extends StatefulWidget {
   /// This property must not be null. It defaults to 8.0 padding on all sides.
   final EdgeInsetsGeometry padding;
 
-
+  /// {@template flutter.material.ExpandIcon.color}
   /// The color of the icon.
   ///
   /// Defaults to [Colors.black54] when the theme's
   /// [ThemeData.brightness] is [Brightness.light] and to
   /// [Colors.white60] when it is [Brightness.dark]. This adheres to the
   /// Material Design specifications for [icons](https://material.io/design/iconography/system-icons.html#color)
   /// and for [dark theme](https://material.io/design/color/dark-theme.html#ui-application)
+  /// {@endtemplate}
```

Comment:
Making the colors Material2 specific and backwards compatible seems OK since ExpansionPanel is no longer part of the Material spec: https://m3.material.io/components. And the M3 spec's icon section doesn't appear to offer any guidelines about color - https://m3.material.io/styles/icons/overview.

@darrenaustin - is there a backwards compatible ColorScheme color that could be user here?

Reply:
Not really. The closest we have is probably `ColorScheme.onSurface`, but even with the default color scheme, it won't match the M2 colors used here exactly.